src/black/__init__.py

```diff
@@ -6144,6 +6196,141 @@ def __str__(self) -> str:
         return ", ".join(report) + "."
 
 
+@dataclass
+class JunitReport:
```

Comment:
I feel like it might be cleaner to add a method to the existing Report class that produces XML output, instead of adding a new class. Would that work?

Reply:
I wanted to leave the existing ```Report``` class untouched.

If you need more Report types in the Future, the ```Report``` class will grow unnecessarily in my opinion.